### DISCUSSION TITLE
Serverless functions run when deployed, locally it returns 502 (EBADF) error.

### DISCUSSION THREAD

**Discussion by Luddzik** [vercel/community#4911] [Help]
### Summary

Hi, I am relatively new to backend development, so any assistance would be appreciated. I have setup basic flutter project with node.js using typescript server project and deployed it to vercel. It has been deployed and it all works fine. 
However, when running locally using 'vercel dev' it runs the flutter frontend page, but accessing any api (using curl/postman/direct url) results in 502: Gateway error, with terminal showing **ERROR: EBADF** without any further details. 

I have tried updating packages, and reinstalling npm packages, but issue still persists. The actual API is the sample .ts file in the example docs, wanted to just establish connection and test it works before adding more complex things. 

I am using MacOS if it affects anything. 
Thank you for help.

### Steps to Reproduce

- run 'vercel dev'
- do any API call (from url/curl/postman..)
- observe return value

**Comment by algology** [comment]
Getting the same issue.
